### DISCUSSION TITLE
Allow to add comment/reason when you suspend a resource

### DISCUSSION THREAD

**Discussion by ThommyH** [fluxcd/flux2#3872] [Proposals]
Sometimes it makes sense to suspend resources/kustomizations. However, more often than not, a resource remains suspended for a long time and resuming it was forgotton or sometimes it is left resumed on purpose. Unfortunately later on you or someone else doesnt fully grasps of why something is suspended in the first place and its unknown if resuming it would be safe.

Therefore I suggest that suspend takes a message and can be shown later.

**Comment by ThommyH** [comment]
we use weave gitops for this now as this supports it out of the box

**Comment by neumantm** [comment]
Relates to https://github.com/fluxcd/flux2/pull/4232
